app/src/main-process/menu/build-default-menu.ts

```diff
@@ -294,10 +298,13 @@ export function buildDefaultMenu(
         click: emit('compare-branch'),
       },
       {
-        label: __DARWIN__ ? 'Create Pull Request' : 'Create &pull request',
+        label: pullRequestLabel,
         id: 'create-pull-request',
         accelerator: 'CmdOrCtrl+R',
-        click: emit('create-pull-request'),
+        click:
+          pullRequestLabel === defaultPullRequestLabel
```

Comment:
Using using-facing strings as the basis for logic makes me nervous. How about we emit a single command (e.g., `show-or-create-pull-request`) and make the decision in the app store based on whether we have a current PR.

Reply:
Decided to move the check into `app.tsx`. I wanted to leave the two functions: `_createPullRequest` and `_showPullRequest` in `app-store.ts` alone since they do two very different things.